test/strategies.js

```diff
@@ -3956,6 +3956,41 @@ const createCurveUsdFLDebtBoostStrategy = () => {
     boostStrategy.addAction(curveUsdCollRatioCheckAction);
     return boostStrategy.encodeForDsProxyCall();
 };
+
+const createCurveUsdPaybackStrategy = () => {
+    const paybackStrategy = new dfs.Strategy('CurveUsdPaybackStrategy');
+
+    paybackStrategy.addSubSlot('&controllerAddr', 'address');
+    paybackStrategy.addSubSlot('&amountToPayback', 'uint256');
+    paybackStrategy.addSubSlot('&crvUsdAddress', 'address');
+
+    const curveUsdHealthRatioTrigger = new dfs.triggers.CurveUsdHealthRatioTrigger(nullAddress, nullAddress, '0');
+    paybackStrategy.addTrigger(curveUsdHealthRatioTrigger);
+
+    const pullTokenAction = new dfs.actions.basic.PullTokenAction(
+        '&crvUsdAddress', // taken from subdata
+        '&eoa',
```

Comment:
I would put the address from which we're pulling the crvUSD token as a subSlot, and then in pullToken pull it from there, thus enabling users who have a curveusd position on their multisig safe to use this strategy to pull crvusd from their EOA